### DISCUSSION TITLE
requirements.txt'

### DISCUSSION THREAD

**Discussion by mutisantoz** [hacksider/Deep-Live-Cam#186] [Q&A]
![aaa](https://github.com/user-attachments/assets/ae0f4ffe-465f-414c-88d7-40900ec5e986)
Boss, I am getting an error like this, what is the solution?

**Comment by GylanSalih** [comment]
Hey i hope your doing well :)
here https://github.com/hacksider/Deep-Live-Cam/discussions/110
a user posted a Install Video https://www.youtube.com/watch?v=MICScLVM1x8
Watch the video maybe you forget a step 
hope i can help you
